README.md

```diff
@@ -22,9 +22,10 @@ on:
 jobs:
   milestone_pr:
     name: attach to PR
+    if: github.event.pull_request.merged == true
```

Comment:
This check happens in the labeler as well, but I agree that it's even better to catch it upfront.